ORM/DataObject.php

```diff
@@ -589,7 +618,7 @@ public function defineMethods() {
 			}
 		}
 
-		if($this->class == 'DataObject') return;
+		if($this->class == 'SilverStripe\ORM\DataObject') return;
```

Comment:
Shorthand, ouch, but okay in this case. Same as above, is_a would maybe be a better way to do it?